.github/workflows/release-arm.yaml

```diff
@@ -1,27 +1,42 @@
----
-name: 'Build arm images'
-
+name: Build arm images
 on:
   push:
     tags:
       - '*'
-
 jobs:
+  get-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          git fetch --prune --unshallow
+      - id: set-matrix
+        run: |
+          content=`cat ./.github/flavors-arm.json`
+          # the following lines are only required for multi line json
+          # the following lines are only required for multi line json
+          # the following lines are only required for multi line json
+          # the following lines are only required for multi line json
```

Comment:
does it need each of these? or just one?

Reply:
saw from the other file it was only once so I assume only one is needed

Reply:
damn it, is the YAML formatter.. it injected those somehow :(

Reply:
if this turns out to be green I'll merge and amend in another PR, guessing maybe there are other pieces to fix too..